twindb_backup/destination/s3.py

```diff
@@ -277,7 +279,7 @@ def _validate_upload(self, object_key):
 
         log.debug("Upload successfully validated")
 
-        return True
+        return 0
```

Comment:
I think the function should return boolean instead of returning numeric error codes as that is more pythonic ;) Let's change all other places.